pact/hft.pact

```diff
@@ -111,52 +65,63 @@
       newbal)
   )
 
-  (defcap ROTATE (token:string account:string)
+  (defcap SUPPLY:bool (id:string supply:decimal)
```

Comment:
Should this replace `UPDATE_SUPPLY`?

Reply:
No, because `update-supply` computes the amount and `UPDATE_SUPPLY` is the private cap to guard it.

Reply:
However this caught a bug which is that `update-supply` is not emitting `SUPPLY`, so I've addressed that. 👍